charts/amazon-cloudwatch-observability/templates/linux/neuron-monitor-daemonset.yaml

```diff
@@ -35,6 +35,8 @@ spec:
         fieldPath: spec.nodeName
   - name: PATH
     value: /usr/local/bin:/usr/bin:/bin:/opt/aws/neuron/bin
+  - name: GOMEMLIMIT
+    value: 160MB
```

Comment:
should this be an int value?

Reply:
Nope - Its a GO runtime variable and it can understand the suffixes -> https://pkg.go.dev/runtime#hdr-Environment_Variables.

Reply:
how did you verify if this env var is getting applied to neuron monitor?

Reply:
Added testing